src/routes/organization/[organizationId]/+page.svelte

```diff
@@ -166,166 +166,187 @@
 <div class="content" style="overflow-x: auto;">
   {#if $organization}
     <div class="row flex-center my-4 max-md:flex-wrap" style="gap: 1rem; align-items: center;">
-      <div class="relative !w-[500px] grow">
-        <div class="card h-fit w-fit sm:!m-0 sm:!ml-auto">
-          Seats Used: {Object.keys($organization.private?.members ?? {}).length}/{$organization.locked.seats}
-        </div>
-      </div>
```

Comment:
If you put back this spacer and have it alternate width at the same time as the photo upload card, it should keep the org title centered as it grows back and forth